### PR TITLE
fixed 2 bugs!

### PR DESCRIPTION
**Pull request by lunfardo314**

### CODE REVIEW

File: packages/vm/core/accounts/internal.go [hunk 1]
```diff
@@ -194,7 +194,7 @@ func DebitFromAccount(state kv.KVStore, agentID *iscp.AgentID, assets *iscp.Asse
 	defer checkLedger(state, "DebitFromAccount OUT")
 
 	if !debitFromAccount(account, assets) {
-		panic(xerrors.Errorf(" agentID: %s. %w", agentID.String(), ErrNotEnoughFunds))
+		panic(xerrors.Errorf(" agentID: %s. %v", agentID.String(), ErrNotEnoughFunds))
```
Comment (dessaya) [inline packages/vm/core/accounts/internal.go:197]: Is this change intentional? `%w` wraps the given error inside ther newly created one.

**Review by dessaya — APPROVED** [review]

File: packages/vm/core/accounts/internal.go (hunk 1 above)
Comment (lunfardo314) [reply packages/vm/core/accounts/internal.go:197]: yes, it is intentional. I explained once: the wrapped panic messages are too scary for any Solo user. So I removed all %w in the code. We may want to revisit the UX of the error messages in more systematic way

Comment (jorgemmsilva) [reply packages/vm/core/accounts/internal.go:197]: yes, but this means we cannot type check the wrapped error (with `errors.Is`)

**Review by jorgemmsilva — APPROVED** [review]
